### PR TITLE
Add physics example

### PR DESCRIPTION
**Pull request by ianvollick**

### CODE REVIEW

File: physics/physics.js [hunk 1]
```diff
@@ -0,0 +1,505 @@
...
+        actor:new CompositorProxy(foil.actor, ['transform'])
+      }
+      message.foils.push(foilProxy);
+    }
+
+    worker.postMessage(message);
+    requestAnimationFrame(tick);
+  };
+
+  if (isMain())
```
Comment (surma) [inline physics/physics.js]: For educational purposes, I’d prefer splitting main thread JS and worker JS into two files. This is just an additional barrier when trying to grok this thing.

File: physics/physics.js [hunk 2]
```diff
@@ -0,0 +1,505 @@
...
+    foil.dy = foil.top;
+    foil.vx = 0;
+    foil.vy = 0;
+
+    return foil;
+  }
+
+  function busySleep(duration) {
+    var start = new Date();
+    while ((new Date()).getTime() - start.getTime() < duration);
```
Comment (surma) [inline physics/physics.js]: Maybe use `window.performance` and rAF to make the jank a little more consistent? See [my jank](https://github.com/GoogleChrome/houdini-samples/blob/master/parallax-scrolling/parallax.js#L31-L35) for parallax scrolling. Yes, I am genuinely reviewing your jank implementation :smile:

**Comment by surma** [comment]
Other than the conglomeration of the worker and main JS, this LGTM :)

File: physics/physics.js (hunk 2 above)
Comment (baptistebriel) [inline physics/physics.js]: Just getting curious here and not sure if I get what you're doing - you're specifically adding some time to the frame in the main thread to produce a jank animation?

Comment (surma) [inline physics/physics.js]: Yup, burning more than 16ms in the main thread should *usually* result in any kind of JS-based animation to jank as we are not reaching 60fps anymore. But not with Houdini :tada:

Comment (baptistebriel) [inline physics/physics.js]: Awesome work. I wanted to do some testing and try Houdini, but instead got this in the DevTools: https://surma.link/things/houdini-intro/devtools.png - Any ideas when a more 'robust' frankenstein build will be out? 😊 Another question, is Houdini's idea based on https://github.com/ianvollick/animation-proxy/ ? Just figured out this has been last updated in 2013.

Comment (surma) [inline physics/physics.js]: As far as I know, we are landing *a subset* of the Houdini specs behind a flag in Canary soon-ish :tm: Don’t know exact dates myself, sorry.

Comment (baptistebriel) [inline physics/physics.js]: No problem, looking forward to it!

**Comment by ianvollick** [comment]
Done, though I didn't make the jank random. I also removed some debugging code I'd left behind.

**Comment by surma** [comment]
Awesome, thanks Ian :)

**Pull request merged**